src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -0,0 +1,38 @@
+// Package launcher configures Lantern to run on system start
+package launcher
+
+import (
+	"github.com/kardianos/osext"
+	"github.com/luisiturrios/gowin"
+
+	"github.com/getlantern/golog"
+)
+
+const (
+	RunDir = `Software\Microsoft\Windows\CurrentVersion\Run`
```

Comment:
nit: it seems like this should just be a local variable in CreateLaunchFile since it's not used anywhere else and just adds to the global state otherwise.